### PR TITLE
Adds long variations of algae, refactors headtail cloths

### PR DESCRIPTION
**Pull request by Llywelwyn**
rscadd: "Added variations to Skrell algae loadout items for longer headtails."
rscadd: "Made Skrell headtail cloths recolourable instead of a set of fixed colours."

made headgear use subtypes properly in the process, and made variations use recolouring instead of having like 58 duplicate sprites for different versions

image

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!review

**Review by RyverStyx — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 1]
```diff
@@ -118,8 +104,11 @@
 	..()
 	var/list/algae = list()
 	algae["glowing algae(dots)"] = /obj/item/clothing/ears/skrell/goop
+	algae["long glowing algae(dots)"] = /obj/item/clothing/ears/skrell/goop/long
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:107]:
```suggestion
	algae["long glowing algae, dots"] = /obj/item/clothing/ears/skrell/goop/long
```

**Review by JohnWildkins — APPROVED** [review]

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 2]
```diff
@@ -118,8 +104,11 @@
 	..()
 	var/list/algae = list()
 	algae["glowing algae(dots)"] = /obj/item/clothing/ears/skrell/goop
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:106]:
```suggestion
	algae["glowing algae, dots"] = /obj/item/clothing/ears/skrell/goop
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 3]
```diff
@@ -118,8 +104,11 @@
...
+	algae["long glowing algae(dots)"] = /obj/item/clothing/ears/skrell/goop/long
 	algae["glowing algae(stripes)"] = /obj/item/clothing/ears/skrell/goop/stripes
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:108]:
```suggestion
	algae["glowing algae, stripes"] = /obj/item/clothing/ears/skrell/goop/stripes
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 4]
```diff
@@ -118,8 +104,11 @@
...
+	algae["long glowing algae(stripes)"] = /obj/item/clothing/ears/skrell/goop/stripes/long
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:109]:
```suggestion
	algae["long glowing algae, stripes"] = /obj/item/clothing/ears/skrell/goop/stripes/long
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 5]
```diff
@@ -118,8 +104,11 @@
...
 	algae["glowing algae(circles)"] = /obj/item/clothing/ears/skrell/goop/circles
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:110]:
```suggestion
	algae["glowing algae, circles"] = /obj/item/clothing/ears/skrell/goop/circles
```

File: code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm [hunk 6]
```diff
@@ -118,8 +104,11 @@
...
+	algae["long glowing algae(circles)"] = /obj/item/clothing/ears/skrell/goop/circles/long
```
Comment (SleepyGemmy) [inline code/modules/client/preference_setup/loadout/loadout_xeno/skrell.dm:111]:
```suggestion
	algae["long glowing algae, circles"] = /obj/item/clothing/ears/skrell/goop/circles/long
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Comment by Llywelwyn** [comment]
failed test is from recurring bug in master, not from pr, im 99.9% sure. rerunning test should fix

**Pull request merged**